packages/console/src/command-history/CommandHistoryViewportUpdater.tsx

```diff
@@ -21,6 +22,8 @@ export type CommandHistoryViewportUpdaterProps = {
   onUpdate: ViewportUpdateCallback<CommandHistoryStorageItem>;
 };
 
+const SET_SEARCH_DEBOUNCE_MS = 500;
```

Comment:
Typically for user input/typing we use 150, or 250. 500 is a bit long. Should make things feel a bit snappier.
```suggestion
const SET_SEARCH_DEBOUNCE_MS = 150;
```

Reply:
Fixed